Lesson19/index.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+
+const MONTH_BIRTHDAY = 'March';
+const FIRST_NAME = 'Marcelo';
+const LAST_NAME = 'Matos';
+
+let city;
+let age;
+let nextCity;
+
+city = 'Winnipeg';
+age = 18;
+nextCity = city;
+
+console.log('Month of birthday: ', MONTH_BIRTHDAY);
+console.log(`Full Name: ${FIRST_NAME} ${LAST_NAME}`);
+
+console.log('City: ', city);
+console.log('Age: ', age);
+console.log('Next City: ', nextCity);
```

Comment:
Really nice that you used best practice for constant names.
Code looks clear and understandable.
Well done!